### DISCUSSION TITLE
How to manual ScaleDown ?

### DISCUSSION THREAD

**Discussion by kzcPo** [argoproj/argo-rollouts#1975] [Q&A]
I want to manual ScaleDown.
`scaleDownDelaySeconds: 0`

I hope so: 
`kubectl argo rollouts scaleDown rollout/rollout-bluegreen2 --revision=3`

**Comment by jinleileiking** [comment]
kubectl scale --replicas=8  rollouts/trans-worker

**Reply by kzcPo** [reply]
I want to manually clean up the old version instead of the current version.
I hope so:
```
kubectl argo rollouts scaleDown rollout/rollout-bluegreen2 --revision=3
```
